Flow.Launcher/App.xaml.cs

```diff
@@ -152,6 +154,20 @@ public static void Main()
 
         #endregion
 
+        #region Notify Icon
+
+        private void InitializeNotifyIcon()
+        {
+            NotifyIcon = new NotifyIcon
+            {
+                Text = Constant.FlowLauncherFullName,
+                Icon = Constant.Version == "1.0.0" ? Launcher.Properties.Resources.dev : Launcher.Properties.Resources.app,
```

Comment:
Is there a way to not have this hardcoded? I am wary when moving to v2.0.0+ we need to bump this dev version too.

Reply:
Yeah ok.

I think it's more consistent if we raise this from 1.0.0 to 2.0.0. we can do this in separate PR.